app/scripts/helpers/price.helpers.js

```diff
@@ -0,0 +1,3 @@
+export function formatPrice(amount, currency) {
+	return `${currency === 'USD' ? '$' : ''}${amount}${currency === 'EUR' ? ' €' : ''}`;
```

Comment:
I personnally don't feel weird to read price in french format in the middle of english text.

The advantage of Intl.NumberFormat is that it works for the placement of the currency, but also for the decimal delimitator.